site-docs/src/main/paradox/dynamodb/usage/fs2.md

```diff
@@ -0,0 +1,51 @@
+# Use with Fs2
+
+## Setup
+You will need these dependencies:
+
+@@dependency[sbt,Maven,Gradle] {
+group="uk.gov.nationalarchives" artifact="da-dynamodb-client_2.13" version=$version$
+}
+
+## Examples
+```scala
+import cats.effect._
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import uk.gov.nationalarchives.DADynamoDBClient
+import uk.gov.nationalarchives.DADynamoDBClient.DynamoDbRequest
+
+val fs2Client = DADynamoDBClient[IO]()
+
+def getAttributeValues(tableName: String, primaryKeyName: String, primaryKeyValue: String, attributeName: String, attributeName2: String): IO[AttributeValue] = {
+  val primaryKeyAttribute = AttributeValue
+    .builder()
+    .s(primaryKeyValue) // '.s' for String type; methods for other types can be found here https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/dynamodb/model/AttributeValue.html#method-detail
+    .build()
+  
+  val dynamoDbRequest = DynamoDbRequest(
+    tableName,
+    Map(primaryKeyName -> primaryKeyAttribute),
+    Map(attributeName -> None, attributeName2 -> None)
+  )
+  fs2Client.getAttributeValue(dynamoDbRequest)
+}
+
+def updateAttributeValues(tableName: String, primaryKeyName: String, primaryKeyValue: String, attributeName: String,
```

Comment:
Can the attribute args not be a single hashmap of `attributeName: newAttributeValue`? The function calls for these could get very long depending on how many attributes need updating.

Reply:
It is, check line 44. The method on line 34 is an example of how someone might write the set up code to call the client; it is not the actual method definition. The method name could have been be anything (I couldn't think of a more creative name); I'll rename it so that it is clearer.

Reply:
Ahhhh, that makes sense. Apologies.